src/site/_data/authorsData.json

```diff
@@ -1094,10 +1094,30 @@
     "github": "brendankenny",
     "image": "image/MtjnObpuceYe3ijODN3a79WrxLU2/sWOfAs4f2WyQMMF9xt1A.jpg"
   },
+  "svenmay": {
+    "country": "DE",
+    "github": "Svenmay",
+    "image": "image/dB6B4Sr8kaaT0KZujRBFC303oFR2/FAG5C2RsJyDw1Ap2DGBw.jpg"
+  },
+  "dstasyev": {
+    "country": "RU",
+    "github": "denisstasyev",
+    "image": "image/dB6B4Sr8kaaT0KZujRBFC303oFR2/GcmQMuJs1yvVmFILU3Xw.jpg"
+  },
   "alexandrawhite": {
      "country": "US",
      "github": "heyawhite",
      "twitter": "heyawhite",
      "image": "image/VbsHyyQopiec0718rMq2kTE1hke2/tKKMrgVB852NNd1UMRzN.png"
-   }
+   },
+  "svenmay": {
+    "country": "DE",
+    "github": "Svenmay",
+    "image": "image/dB6B4Sr8kaaT0KZujRBFC303oFR2/FAG5C2RsJyDw1Ap2DGBw.jpg"
+  },
+  "dstasyev": {
```

Comment:
Not sure about two things:
- If possible, then I would prefer to use `denisstasyev` as username
- Please check it looks like a crooked merge - information about authors is duplicated

Reply:
No prob, that is fixed now.